gap/OrbitalGraphs.gi

```diff
@@ -81,10 +81,104 @@ end);
 
 InstallMethod( OrbitalClosure, "for a permutation group",
                [ IsPermGroup ],
-G -> Intersection(List(OrbitalGraphs(G), AutomorphismGroup)) );
 # TODO: TwoClosure as implemented by Heiko Theißen requires the group
-#       to be transitive.
+# to be transitive.
```

Comment:
Why?

Reply:
No particular reason. It's just whitespace in a comment, didn't think about it much.